server/ladder_service.py

```diff
@@ -52,15 +52,13 @@ def __init__(
         self._informed_players: Set[Player] = set()
         self.game_service = game_service
 
-        # Fallback legacy map pool and matchmaker queue
-        self.ladder_1v1_map_pool = MapPool(0, "ladder1v1")
+        # Matchmaker queue
         self.queues = {
             'ladder1v1': MatchmakerQueue(
                 game_service,
                 name="ladder1v1",
                 featured_mod=FeaturedModType.LADDER_1V1,
-                rating_type=RatingType.LADDER_1V1,
-                map_pools=[(self.ladder_1v1_map_pool, None, None)]
+                rating_type=RatingType.LADDER_1V1
```

Comment:
Like I said, I think we should remove this whole hardcoded queue. There is no way to use the new map pool format without inserting this queue into the database, so there's no reason to keep it around.

That will also mean that the `TODO` in `initialize` can be resolved, since there wont be any more hardcoded queues to initialize.

Reply:
Removing both of those generates a slew of crashes in multiple unit and integration tests but if those just expect the existing information from the hardcoded queue I'll change those.

Reply:
Which tests are failing? Why? Are they tests that are specifically checking for the hardcoded queue? Then they can be deleted too.